cli/run.go

```diff
@@ -81,7 +81,9 @@ func run(c *cli.Context) {
 		log.Debugln("[ENVMAN] - Executing command:", cmdToExecute)
 
 		if exit, err := runCommandModel(cmdToExecute); err != nil {
-			log.Error("[ENVMAN] - Failed to execute command:", err)
```

Comment:
or maybe, print it with `.Error` if `exit == 0` ? -> ex: in case of the `bash` arguments list too long issue?